strategy/from_json/to_serializable_test.ts

```diff
@@ -104,7 +104,7 @@ test({
     }
     class Test extends Serializable {
       @SerializeProperty({
-        fromJSONStrategy: (v) => new OtherClass().fromJSON(v),
+        fromJSONStrategy: (v) => (v as JSONArray).map(el => new OtherClass().fromJSON(el)),
```

Comment:
so you can type this at the param level and it should apply to the function scope